actions/upload_sessions.go

```diff
@@ -18,15 +18,22 @@ type UploadSessionResource struct {
 // Request Response structs
 
 type uploadSessionCreateReq struct {
-	GenesisHash   string `json:"genesisHash"`
-	FileSizeBytes int    `json:"fileSizeBytes"`
-	BetaIP        string `json:"betaIp"`
+	GenesisHash          string `json:"genesisHash"`
+	FileSizeBytes        int    `json:"fileSizeBytes"`
+	BetaIP               string `json:"betaIp"`
+	StorageLengthInYears int    `json:"storageLengthInYears"`
```

Comment:
Should we store the storage length in the DB too? Could be helpful if we need to debug in the future.

Reply:
I'll add it